jenkins-ci/Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+pipeline {
+    agent any
+    stages {
+        stage('all-in-one: clean environment') { 
+            steps { 
+                sh 'oc get all'
```

Comment:
nit you can create a namespace(project) and delete it after the test. It might be better because  it do not depend on a specific delete command.